src/cocoa_joystick.m

```diff
@@ -399,11 +399,13 @@ int _glfwPlatformPollJoystick(_GLFWjoystick* js, int mode)
 
             const long delta = axis->maximum - axis->minimum;
             if (delta == 0)
-                _glfwInputJoystickAxis(js, i, 0.f);
+            {
+                _glfwInputJoystickAxis(js, (int) i, 0.f);
+            }
```

Comment:
Please don't add braces for a single unbroken line.

Reply:
Ah sorry, just force of habit.